plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -207,17 +215,43 @@ app_get_flatpak_ref_name (GsApp *app)
 	return gs_app_get_metadata_item (app, "flatpak::RefName");
 }
 
+static gchar **
+get_default_locales (GsPlugin *plugin)
+{
+	g_auto(GStrv) flatpak_default_locales = NULL;
+	g_autoptr(GError) local_error = NULL;
+
+	GsPluginData *priv = gs_plugin_get_data (plugin);
+	if (priv->installation == NULL)
+		return NULL;
+
+	flatpak_default_locales = flatpak_installation_get_default_languages (priv->installation, &local_error);
+	if (local_error != NULL) {
+		g_warning ("No user locales could be retrieved! %s", local_error->message);
+		return NULL;
+	}
+
+	return g_steal_pointer (&flatpak_default_locales);
+}
+
 static gboolean
 gs_plugin_locale_is_compatible (GsPlugin *plugin,
 				const char *locale)
```

Comment:
```suggestion
				const char *app_locale)
```

(I’m concerned that almost every variable in this function is named `locale`-something, and it’s getting a bit confusing what each variable is — is `locale` the locale from the app, or from flatpak, or from the user’s environment?? I realise this was a problem before, but we might as well fix it while we’re here.)